snafu/flent_wrapper/Dockerfile

```diff
@@ -8,8 +8,10 @@ RUN mkdir /tmp/install
 WORKDIR /root
 RUN wget https://github.com/HewlettPackard/netperf/archive/netperf-2.7.0.tar.gz && tar -xzf netperf-2.7.0.tar.gz
 WORKDIR /root/netperf-netperf-2.7.0
+# Workaround: We need to download latest version of config.guess to make configure able to detect other CPU architectures correctly
```

Comment:
Nice catch! I ran into this as well when debugging some of these builds. Glad to see this was included :)